CONTRIBUTING.md

```diff
@@ -25,7 +25,20 @@ Happy coding!
 
 ## Netlify
 
-We use [Netlify](https://www.netlify.com/) in the background to build our page:
+We use [Netlify](https://www.netlify.com/) in the background to build and deploy our pages.
 
-- Commits to master trigger a build of the live page.
-- Pull requests trigger a build of the related branch.
+### Branching
+Our branching strategy is the following:
+- `master` is the Production branch
+- `develop` is the branch for current development. Any feature starts from it and is merged back to it once approved.
+- a `feature/[issue_id]-[short_description]` branch is created whenever a new feature is started
+- a `bugfix/[issue_id]-[short_description]` branch is created whenever a new bugfix is started
+- hotfixes could processed like normal features or directly merged into `master` and deployed right away
```

Comment:
@backflip @ralf57 This is a big red flag. Every change should be sourced from develop and targeted to develop, alternatively cherry-picked into master. But a change should never go only into master without a merge to develop. This will produce big merge conflicts and destroy any attempt for a clean commit history once someone want to merge the complete development branch back into master. That developer will be confronted with a diverged state of master and develop.

Reply:
Also a very valid point, makes sense to me. @ralf57, how about changing `or directly merged` to `or additionally cherry-picked`?

Reply:
@backflip @renestalder I don't think it's an issue to directly merge hotfixes to `master`.
Since `master` always represents the clean state, it can be merged into `develop` at any time, pulling in also the hotfixes

Reply:
@ralf57 I would really recommend at least following gitflow on that one and merge into both branches, master and develop. I don't agree on the part that there will be no issues if two branches have a huge diverged history, both frequently getting commits. Merging in one direction will always be easier than trying to syncing two histories bi-directional.

Reply:
@renestalder honestly I fail to see the issue with merging hotfixes directly into `master`.
As long as you immediately merge `master` into `develop` the risk of merge conflicts is minimal, not higher than with other branching strategies.